src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/SingleServerDelegatingEndpointTestBase.cs

```diff
@@ -138,12 +137,19 @@ public override Task OnInitializedAsync(VSInternalClientCapabilities clientCapab
                 RazorLanguageServerCustomMessageTargets.RazorSpellCheckEndpoint => await HandleSpellCheckAsync(@params),
                 RazorLanguageServerCustomMessageTargets.RazorDocumentSymbolEndpoint => await HandleDocumentSymbolAsync(@params),
                 RazorLanguageServerCustomMessageTargets.RazorProjectContextsEndpoint => await HandleProjectContextsAsync(@params),
+                RazorLanguageServerCustomMessageTargets.RazorMethodEndpointName => HandleSimplifyMethod(@params),
                 _ => throw new NotImplementedException($"I don't know how to handle the '{method}' method.")
             };
 
             return (TResponse)result;
         }
 
+        private static TextEdit[] HandleSimplifyMethod<TParams>(TParams @params)
+        {
+            Assert.IsType<DelegatedSimplifyMethodParams>(@params);
+            return null;
```

Comment:
I tried returning the text edit that was passed in as a hardcoded response, but got grief with the razor formatter and have been unable to debug tests for some reason :/... Right now the tests do not take simplification into account at all, but there is a test that checks the simplification works as expected in the roslyn repo.